src/bullsandcows/Main.java

```diff
@@ -1,7 +1,194 @@
-package bullsandcows;
+
+import java.util.*;
+//package bullsandcows;
 
 public class Main {
+
+    public static Set<Character> secretGenerator(int numbers, int possibleNumbers) {
+        /*
+        We will generate ints and then convert them to valid characters from ascii table.
```

Comment:
It may be better to use /** */ before the method's name. It is Java-way style. See here: 

```
/**
 * It finds something in the given collection. The method does not produce side-effects.
 */
public Something findSomething(Collection<E> collection)
```